test/integration/env_test.js

```diff
@@ -1,25 +1,61 @@
-suite('setting env variables', function() {
-  var co = require('co');
-  var testworker = require('../post_task');
+import assert from 'assert';
+import testworker from '../post_task';
```

Comment:
hrm, I thought `import` statements were only allowed at the beggining of the file.